Src/Support/Google.Apis.Auth.Tests/OAuth2/GoogleCredentialTests.cs

```diff
@@ -846,5 +876,23 @@ public Task RevokeTokenAsync(string userId, string token, CancellationToken task
 
             public bool ShouldForceTokenRetrieval() => throw new NotImplementedException();
         }
+
+        /// <summary>
+        /// Creates json parameters for an impersonated credential with the specified scopes.
+        /// </summary>
+        private static JsonCredentialParameters CreateImpersonatedCredentialParameters(string[] scopes) =>
```

Comment:
To match the rest of the tests here, and also to test that we are indeed reading scopes from JSON, create a new FakeImpersonatedScopedCredential constant with what would be the JSON content, and use that on the tests instead of this method.